types/webidl-conversions/index.d.ts

```diff
@@ -1,17 +1,15 @@
-// Type definitions for webidl-conversions 6.1
+// Type definitions for webidl-conversions 7.0
 // Project: https://github.com/jsdom/webidl-conversions#readme
 // Definitions by: ExE Boss <https://github.com/ExE-Boss>
+//                 BendingBender <https://github.com/BendingBender>
```

Comment:
Same question as your other PR; are you looking to be come an owner/reviewer, or just authorship credit? https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

Reply:
Yes, that's what I wanted.